Makefile

```diff
@@ -14,7 +14,7 @@ clean:
 	$(REBAR) clean
 	rm -rf _build _etylizer rebar.lock
 
-test: build testtest
+test: build #testtest
```

Comment:
`testtest` runs the unit tests for the tests.

Background: when writing programs that should type check (as a unit test for the type checker), the program sometimes does not behave as the author of the test intended. Hence, we write unit tests to check that the tests does what it is supposed to do.